torchvision/transforms/v2/_color.py

```diff
@@ -54,6 +54,20 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
         return self._call_kernel(F.rgb_to_grayscale, inpt, num_output_channels=params["num_input_channels"])
 
 
+class RGB(Transform):
+    """Converts images or videos to RGB (if they are already not RGB)
```

Comment:
Sorry, forgot that one (a Python-wide convention)

https://peps.python.org/pep-0257/#one-line-docstrings

> The docstring is a phrase ending in a period. It prescribes the function or method’s effect as a command (“Do this”, “Return that”), not as a description; e.g. don’t write “Returns the pathname …”.

```suggestion
    """Convert images or videos to RGB (if they are already not RGB)
```